cpp/src/Glacier2/RouterI.cpp

```diff
@@ -201,11 +193,6 @@ Glacier2::RouterI::updateObserver(const shared_ptr<Glacier2::Instrumentation::Ro
     // Can only be called with the SessionRouterI mutex locked
 
     _observer = _routingTable->updateObserver(observer, _userId, _connection);
-    _clientBlobject->updateObserver(_observer);
-    if (_serverBlobject)
-    {
-        _serverBlobject->updateObserver(_observer);
-    }
```

Comment:
What was this about?

Reply:
Not sure the observer logic. Here, it's just a mechanical removal - I removed Blobject::updateObserver because it had only an effect in buffered mode:

```cpp
void
Glacier2::Blobject::updateObserver(const shared_ptr<Glacier2::Instrumentation::SessionObserver>& observer)
{
    if (_requestQueue)
    {
        _requestQueue->updateObserver(observer);
    }
```